### PR TITLE
Build Product page

### PR DESCRIPTION
**Pull request by ishitakeshawani**
Preview link:  https://deploy-preview-2--stunnershub.netlify.app/

### CODE REVIEW

File: pages/products/products.css [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+}
+
+.rating-title {
+  margin-bottom: 1rem;
+}
+
+.sort-by-title {
+  margin-top: 1rem;
+  margin-bottom: 1rem;
+}
```
Comment (anupkgurung) [inline pages/products/products.css:57]: try usig shorthand.

**Review by anupkgurung — APPROVED** [review]
let a review please try to make changes whenever possible. Rest is good.

File: pages/products/products.css (hunk 1 above)
Comment (ishitakeshawani) [reply pages/products/products.css:57]: yes

**Review by rohanmathur91 — APPROVED** [review]
Looks good to me

**Pull request merged**